Adds info message that 'groff-x11' package might be missing when executing
'groff -X' and 'gxditview' is not found.

Author: Jan Vcelak <jvcelak@redhat.com>
Fixes: #530788 groff and groff-gxditview shouldn't be split

diff -ur groff-1.20.1.orig/src/roff/groff/pipeline.c groff-1.20.1/src/roff/groff/pipeline.c
--- groff-1.20.1.orig/src/roff/groff/pipeline.c	2010-03-19 11:06:37.437182240 +0100
+++ groff-1.20.1/src/roff/groff/pipeline.c	2010-03-19 11:42:14.820161806 +0100
@@ -486,6 +486,8 @@
       execvp(commands[i][0], commands[i]);
       error("couldn't exec %1: %2",
 	    commands[i][0], strerror(errno), (char *)0);
+	  if (strcmp(commands[i][0], "gxditview") == 0)
+	    fprintf(stderr, "You might be missing 'groff-x11' package.\n");
       fflush(stderr);			/* just in case error() doesn't */
       _exit(EXEC_FAILED_EXIT_STATUS);
     }
